spring-cloud-modules/pom.xml

```diff
@@ -52,7 +52,7 @@
         <module>spring-cloud-bus</module>
         <module>spring-cloud-data-flow</module>
         <module>spring-cloud-sleuth</module>
-        <module>spring-cloud-openfeign-2</module>
+        <!--<module>spring-cloud-openfeign-2</module>--> <!-- failing -->
```

Comment:
please revert this change, openfeign-2 should be fixed in the master branch. please update from master and PR should be running fine.